Gopkg.lock

```diff
@@ -789,6 +789,7 @@
     "github.com/operator-framework/operator-sdk/pkg/ready",
     "github.com/operator-framework/operator-sdk/version",
     "github.com/pkg/errors",
+    "gopkg.in/yaml.v2",
```

Comment:
why it is needed?

Reply:
removed https://github.com/percona/percona-xtradb-cluster-operator/pull/418/commits/c38f1fc0f97c48e615fca431ae5db41561f42adf